### PR TITLE
fix nce, multinomial, Categorical, Normal, Uniform en doc

### PR DESCRIPTION
**Pull request by pangyoki**
### PR types

Others

### PR changes

APIs

### Describe

fix en doc for apis

## paddle.static.nn.nce

- code_example:['nce_en_1.py']   ->   add `paddle.enable_static()` in sample code
- has_fluid:True   ->   use `paddle.xxx` instead of `fluid.xxx`
- has_variable:True   ->   change `Variable` to `Tensor`

图片
图片

## paddle.multinomial
- has_print:True   ->   delete `x.numpy()` in `print`

图片

## paddle.distribution.Categorical
- has_print:True   ->   delete `x.numpy()` in `print`

图片

## paddle.distribution.Normal
- has_disable_static   ->   delete `disable_static()`

图片

## paddle.distribution.Uniform
- has_disable_static   ->   delete `disable_static()`

图片

### CODE REVIEW

**Review by TCChenlong — DISMISSED** [review]
LGTM

File: python/paddle/distribution.py [hunk 1]
```diff
@@ -201,7 +201,6 @@ class Uniform(Distribution):
           import paddle
           from paddle.distribution import Uniform
 
-          paddle.disable_static()
           # Without broadcasting, a single uniform distribution [3, 4]:
           u1 = Uniform(low=3.0, high=4.0)
           # 2 distributions [1, 3], [2, 4]
```
Comment (zhiqiu) [inline python/paddle/distribution.py:205]: In most cases, no need to use numpy in the sample code.

File: python/paddle/distribution.py [hunk 2]
```diff
@@ -672,13 +670,13 @@ class Categorical(Distribution):
...
             x = paddle.rand([6])
-            print(x.numpy())
+            print(x)
             # [0.5535528  0.20714243 0.01162981
             #  0.51577556 0.36369765 0.2609165 ]
 
             paddle.seed(200) # on CPU device
             y = paddle.rand([6])
-            print(y.numpy())
+            print(y)
```
Comment (zhiqiu) [inline python/paddle/distribution.py:675]: The format of `print(y)` is not right. Same for other places.

Comment (pangyoki) [reply python/paddle/distribution.py:675]: done

File: python/paddle/distribution.py (hunk 1 above)
Comment (pangyoki) [reply python/paddle/distribution.py:205]: done

File: python/paddle/distribution.py (hunk 2 above)
Comment (pangyoki) [reply python/paddle/distribution.py:675]: dont't need to modify

**Review by zhiqiu — APPROVED** [review]

**Review by TCChenlong — APPROVED** [review]
LGTM

**Pull request merged**